string filename
uint32 duration
---
bool result
